moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -141,7 +166,7 @@ const ComponentForm = ({
             startIcon={<CheckCircle />}
             type="submit"
           >
-            {formButtonText}
+            {isSignalComponent ? "Save" : formButtonText}
```

Comment:
In the first two sets of bullet points, I think the intent is to have this button's content text say `Save Signal Component`, but I'm not sure that it's getting set. I do see that it's set to say "Continue" on line 91 of `src/views/projects/projectView/ProjectComponents/CreateComponentModal.js`. Here is what I see in this case:

![image](https://user-images.githubusercontent.com/3653206/211705058-2af41316-e325-4d1d-a4e2-aed59a6d91ea.png)

Reply:
This is actually supposed to say "Save". I updated from some of John's feedback, and I forgot to update the test instructions. Gotta get better at sweeping through test instructions after updates like this. 😅 Thanks for looking out for this! 🙏